crates/bevy_utils/src/label.rs

```diff
@@ -98,6 +98,14 @@ macro_rules! define_boxed_label {
                 self.dyn_clone()
             }
         }
+
+        impl ScheduleLabel for Box<dyn ScheduleLabel> {
```

Comment:
Why does this say ScheduleLabel instead of using the macro parameters?

Reply:
Yeah, this is definitely a mistake.